pkg/api/validation/validation_test.go

```diff
@@ -3506,6 +3506,11 @@ func TestValidatePodSpec(t *testing.T) {
 	minGroupID := types.UnixGroupID(0)
 	maxGroupID := types.UnixGroupID(2147483647)
 
+	err := utilfeature.DefaultFeatureGate.Set("PodPriority=true")
```

Comment:
not sure how setting globals will affect tests that run in parallel, but at least obtain and defer restoring the original value